packages/cli/src/cliEntry.js

```diff
@@ -93,26 +91,18 @@ const addCommand = (command: CommandT, ctx: ContextT) => {
   const options = command.options || [];
 
   const cmd = commander
-    .command(command.name, undefined, {
-      noHelp: !command.description,
-    })
+    .command(command.name, undefined, {noHelp: !command.description})
     .description(command.description)
     .action(function handleAction(...args) {
       const passedOptions = this.opts();
       const argv: Array<string> = Array.from(args).slice(0, -1);
 
       Promise.resolve()
-        .then(() => {
-          assertRequiredOptions(options, passedOptions);
```

Comment:
`commander` validation actually works and commands fail when required arguments are not passed, so this fn was never called?

Reply:
This one checks for required options, see the linked issue in `assertRequiredOptions.js` file. It allows to require options, when the: "required: true" is set.

Reply:
I don't think we use `required: true` in any of the options for built-in commands, but it's a breaking change (potentially) for other users that provide 3rd party commands. Would consider leaving it as it's a nice feature - we can always work upstream.

Reply:
Oh didn't realise that :<

Reply:
You definitely need to document stuff like that 😛

Reply:
It's all documented in the `assertRequiredOptions` file description! I wouldn't remember that myself otherwise

Reply:
I mean documenting plugins to CLI and how they work

Reply:
Yeah, CLI was historically missing some documentation, but it's finally improving!